engine/keep.go

```diff
@@ -0,0 +1 @@
+package engine
```

Comment:
What's the purpose of this file?

Reply:
same as https://github.com/ledgerwatch/interfaces/blob/master/remote/keep.go